doc/classes/PhysicsServer3D.xml

```diff
@@ -1066,6 +1081,14 @@
 				Sets the value for a space parameter. A list of available parameters is on the [enum SpaceParameter] constants.
 			</description>
 		</method>
+		<method name="space_step" is_experimental="true">
+			<return type="void" />
+			<param index="0" name="space" type="RID" />
+			<param index="1" name="delta" type="float" />
+			<description>
+				Steps [param space]. This can be used to create a rollbackable simulation. You should call [method space_flush_queries] first.
```

Comment:
Same here, I don't like "rollbackable", especially since this doesn't actually directly allow rolling back the simulation, but rather only manually stepping it forward after a rollback.

> This allows for advanced rollback-style networking, or predicting the outcome of hitting a ball in a billiards game.

Reply:
Proposal:

> Manually advance the physics simulation forward in time. This technique can be used for speeding up physics simulations, as seen in advanced rollback-style networking, or for predicting outcomes in scenarios such as hitting a ball in a billiards game.

Reply:
Disagree. Just describe it function, and provide an example optionally.

Reply:
See [here](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html) for pointers on phrasing

Reply:
My problem is that "rollbackable" isn't a word. It comes across as janky and feels especially "made-up" (because it is). It's also meaningless unless you already know the context that it's used in networking. "Rollback-style networking" at least gives people something to google if they've never heard of it before.

I like @tedbarth's proposal the best:

>Manually advance the physics simulation forward in time. This technique can be used for speeding up physics simulations, as seen in advanced rollback-style networking, or for predicting outcomes in scenarios such as hitting a ball in a billiards game.